AmplifyPlugins/Auth/AWSAuthPlugin/Models/Options/AWSAuthWebUISignInOptions.swift

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct AWSAuthWebUISignInOptions {
+
+    public let idpIdentifier: String?
```

Comment:
Add docs for these properties; especially `idpIdentifier` which we decided to leave named this way so it would map to the underlying Cognito property and make it easier for customers to search for.